Makefile

```diff
@@ -11,6 +11,10 @@ clean-kubernetes:
 	- kubectl delete -n openfaas-fn deploy/env-test-labels || : ; kubectl delete -n openfaas-fn svc/env-test-labels || :
 	- kubectl delete -n openfaas-fn deploy/env-test-verbs  || :; kubectl delete -n openfaas-fn svc/env-test-verbs || :
 	- kubectl delete -n openfaas-fn deploy/test-secret  || :; kubectl delete -n openfaas-fn svc/test-secret || :
+	- kubectl delete -n openfaas-fn deploy/test-auto-scaling  || :; kubectl delete -n openfaas-fn svc/test-auto-scaling || :
```

Comment:
Could we be lazy with our clean-up and delete all service / deploy and secret entries in the namespace?

Reply:
i.e. `kubectl delete -n openfaas-fn deploy --all`

Reply:
I think this would be a good idea, I didn't realize I had to add my function to this delete block until I looked at another PR.